aepsych/models/monotonic_projection_gp.py

```diff
@@ -173,7 +163,7 @@ def posterior(
             # using numpy because torch doesn't support vectorized linspace,
             # pytorch/issues/61292
             grid: Union[np.ndarray, torch.Tensor] = np.linspace(
-                self.lb[dim],
+                self.inducing_points.min(dim=0).values,
```

Comment:
This doesn't seem equivalent to before.

Reply:
Solved with using the lb and dim from inducing points method